jsonnet/kube-prometheus/prometheus/prometheus.libsonnet

```diff
@@ -178,7 +178,8 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
         spec: {
           replicas: p.replicas,
           version: $._config.versions.prometheus,
-          image: $._config.imageRepos.prometheus + ':' + $._config.versions.prometheus,
+          image: $._config.imageRepos.prometheus,
+	  tag: $._config.versions.prometheus,
```

Comment:
we should just remove this I think, then it should all work with specifying the tag in the `image`, as ultimately we want to deprecate `tag`, `sha` and `baseImage`

Reply:
Done, can you have another look, thanks!